src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    public static final int FIGURE_TYPES = 5;
+    public static final int SIDE_MAX = 10;
+
+    public Figure getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int figureNumber = (int) Math.floor(Math.random() * FIGURE_TYPES);
+        int sideA = (int) Math.floor(Math.random() * SIDE_MAX + 1);
+        int sideB = (int) Math.floor(Math.random() * SIDE_MAX + 1);
+        int sideC = (int) Math.floor(Math.random() * SIDE_MAX + 1);
+        Figure figure;
+        switch (figureNumber) {
+            case 0:
+                figure = new Square(sideA,colorSupplier.getRandomColor());
```

Comment:
```suggestion
                return new Square(sideA,colorSupplier.getRandomColor());
```
давай одразу будемо повертати значення